src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java

```diff
@@ -14,11 +16,16 @@ public ArrayCollection(int capacity){
     }
 
     public Object get(int i){
+        Objects.checkIndex(i, capacity);
         return content[i];
     }
 
     public boolean isEmpty(){
         int length = 0;
+        Object[] obj = content;
```

Comment:
obj variable is redundant
obj.length == 0 is always false
it is more preferable to use java.util.Objects::isNull method instead of 'obj == null'